### PR TITLE
Docs: updated codeowners file

### PR DESCRIPTION
**Pull request by lwandz13**
Under Technical documentation, I removed myself from data sources, and added  myself to admin (IAM)

### CODE REVIEW

**Comment by baldm0mma** [comment]
Looks like it's mad about the `blem` in the dockerfile?
```
Stdout:

Stderr:
#0 building with "default" instance using docker driver

#1 [internal] load build definition from Dockerfile
#1 transferring dockerfile: 5.80kB done
#1 DONE 0.0s
Dockerfile:1
--------------------
   1 | >>> blem# syntax=docker/dockerfile:1
   2 |     
   3 |     ARG BASE_IMAGE=alpine:3.18.5
--------------------
ERROR: failed to solve: dockerfile parse error on line 1: unknown instruction: blem#
```

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# syntax=docker/dockerfile:1
+blem# syntax=docker/dockerfile:1
```
Comment (briangann) [inline Dockerfile:1]: this appears to be unintended extra text

**Review by jdbaldry — APPROVED** [review]
Thanks Larissa, I've removed the accidental changes and reformatted the section but kept your CODEOWNERs change from datasources to administration.

File: Dockerfile (hunk 1 above)
Comment (lwandz13) [reply Dockerfile]: this has been removed - thank you for catching it!

**Comment by baldm0mma** [comment]
@lwandz13 looks like the `continuous-integration/drone/pr` check got stuck on a flaky loki test. I restarted the drone run and it passed! I think you're good to go to merge. :)

**Pull request merged**